package.json

```diff
@@ -168,13 +168,13 @@
     "@types/uuid": "^3.4.3",
     "@vscode/extension-telemetry": "^0.6.2",
     "fs-extra": "^4.0.3",
-    "npm": "^8.13.1",
```

Comment:
This was never required

Reply:
No idea why we even add npm as a dependency,
I've tested the generated VSIX  (generated from az piplines) & works as expected.